phpunit/class-wp-rest-block-pattern-categories-controller-test.php

```diff
@@ -1,117 +0,0 @@
-<?php
-/**
- * Unit tests covering WP_REST_Block_Pattern_Categories_Controller functionality.
- *
- * @package WordPress
- * @subpackage REST_API
- */
-
-/**
- * Unit tests for REST API for Block Pattern Categories.
- *
- * @group restapi
- * @covers WP_REST_Block_Pattern_Categories_Controller
- */
-class WP_REST_Block_Pattern_Categories_Controller_Test extends WP_Test_REST_Controller_Testcase {
```

Comment:
This function is not in Gutenberg anymore. cc @jsnajdr  for confirmation. Let's also confirm that the tests were backported properly.

Reply:
This was backported to WordPress 6.0 as part of #39889.